src/clients/lan_auto.rb

```diff
@@ -141,6 +141,7 @@ def main
         end
         @new = FromAY(@param)
         Lan.Import(@new)
+        LanUdevAuto.Import(@new)
```

Comment:
Interesting. I wonder how this was supposed to work before:
 https://github.com/yast/yast-autoinstallation/blob/3e3cc69b8dd7129f2efa7d31f805ce7980120818/src/clients/inst_autosetup.rb#L354
 see also https://bugzilla.novell.com/show_bug.cgi?id=303916